packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -135,14 +153,9 @@ String generateTestBootstrap({
 import 'dart:convert';
 import 'dart:io';  // ignore: dart_io_import
 
-// We import this library first in order to trigger an import error for
-// package:test (rather than package:stream_channel) when the developer forgets
-// to add a dependency on package:test.
-import 'package:test/src/runner/plugin/remote_platform_helpers.dart';
```

Comment:
Note with my proposed changes this file basically stays the same. We will now depend on the sources from test_api and test_core though.